src/luascript.cpp

```diff
@@ -8742,10 +8764,20 @@ int LuaScriptInterface::luaPlayerSendOutfitWindow(lua_State* L)
 
 int LuaScriptInterface::luaPlayerAddMount(lua_State* L)
 {
-	// player:addMount(mountId)
+	// player:addMount(mountId or mountName)
```

Comment:
Since you are changing a function I would say yes.
Mark ones told me to do it, since if we get a problem related to this function we should be able to search for "mount(s)" and not have to check every PR.